core/iwasm/aot/aot_loader.c

```diff
@@ -522,6 +523,56 @@ load_target_info_section(const uint8 *buf, const uint8 *buf_end,
         return false;
     }
 
+    /* Check feature flags */
```

Comment:
Had better add a function to check the flags, or the code may be too long here:
```C
static bool
check_feature_flags(uint32 feature_flags);
```

Reply:
OK